src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -138,6 +133,25 @@ public class EvtItem extends SkriptEvent {
 					.since("2.8.0")
 					.requiredPlugins("Paper 1.16+");
 		}
+
+		Skript.registerEvent("Brewing Complete", EvtItem.class, BrewEvent.class, "brew[ed] [complete[d]|finish[ed]] [of %-itemtypes%]")
```

Comment:
```suggestion
		Skript.registerEvent("Brewing Complete", EvtItem.class, BrewEvent.class, "brew[ed] [complete[d]|finish[ed]] [(of|for) %-itemtypes%]")
```
Might be worth splitting this into multiple patterns, it's getting rather complicated in 1. 
Also, thoughts about `completion`?